source/js/components/newsletter-signup/organisms/blog-body-signup.jsx

```diff
@@ -198,7 +198,14 @@ class BlogBodySignForm extends Component {
 
   render() {
     return (
-      <div className="tw-relative tw-border tw-px-8 tw-pt-14 tw-pb-12 medium:tw-p-16 before:tw-absolute before:tw-top-0 before:tw-left-1/2 before:-tw-translate-x-1/2 before:-tw-translate-y-1/2 before:tw-content-[''] before:tw-inline-block before:tw-w-[72px] before:tw-h-14 before:tw-bg-[url('../_images/glyphs/letter.svg')] before:tw-bg-white before:tw-bg-no-repeat before:tw-bg-center before:tw-bg-[length:24px_auto]">
+      <div
+        className={`
+          ${this.props.innerWrapperClass}
```

Comment:
This is the only change introduced to this file. The rest of the existing class names stay the same.